### ISSUE TITLE
Discuss best location for result/output files

### ISSUE THREAD

**Issue by tobyhodges** [brownsarahm/python-data-project#55]
Several CC@Home2020 participants were keen to learn good practice for where to put results/outputs files (inside or outside 'data' folder?), whether to put them under version control, etc. 

The material could be improved by adding discussing this somehwere - maybe episode 1, or maybe later in the lesson if more appropriate. This could include mention of using .gitignore to prevent certain intermediate/results files from being included in the repository e.g. if very large.

**Comment by vinisalazar** [comment]
I believe this could be inserted in Episode 02. On the .gitignore section, there is a sentence that explicitly says:

> [...] Results should be ignored.

I could push a commit to #75 (or make a separate PR) expanding the .gitignore explanation and adding a `mkdir results` command in the code block at the start of the lesson.

Best,
Vini

**Comment by brownsarahm** [comment]
I agree that's a good place to add this material, but in a separate PR
